CMakeLists.txt

```diff
@@ -25,6 +25,14 @@
 ################################################################################
 cmake_minimum_required( VERSION 3.5 )
 
+macro(set_var_to_condition var)
+    if(${ARGN})
+        set(${var} TRUE)
+    else()
+        set(${name} FALSE)
```

Comment:
This is wrong, `${var}` should be used here. No problem so far because unset var is evaluated to FALSE in conditionals, _and_ we do not use `set_var_to_condition` to **_change_** values of variables. To be fixed soon.